db/migrate/20210828205123_create_users.rb

```diff
@@ -0,0 +1,16 @@
+class CreateUsers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :email
+      t.string :slack_handle
+      t.string :github_handle
+      t.string :working_styles, :array => true, :default => []
+      t.string :cohort
+      t.string :pronouns
+      t.string :github_access_token
```

Comment:
Schema here looks great. I'm assuming this github_access_token is something we need to include on the front end as well?

Reply:
You won't need to store it, but we will might be sending it back and forth...it depends on how we implement auth.